dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/handler/ExternalSourceEntryItemUriListHandler.java

```diff
@@ -51,11 +46,9 @@
     private CollectionService collectionService;
 
     private static final Logger log = org.apache.logging.log4j.LogManager
-        .getLogger(ExternalSourceEntryUriListHandler.class);
+        .getLogger(ExternalSourceEntryItemUriListHandler.class);
 
-
-    @Override
-    public boolean supports(List<String> uriList, String method) {
+    public boolean supports(List<String> uriList, String method, Class clazz) {
```

Comment:
I think you lost the `@Override` annotation here.

Reply:
This has been updated